src/routes/RemediationDetailsComponents/ActionsContent/SystemsModal/SystemsModal.js

```diff
@@ -68,7 +68,7 @@ SystemsModal.propTypes = {
 
 const SystemsModalContentProvider = (props) => {
   return (
-    <TableStateProvider>
+    <TableStateProvider isNewContext={true}>
```

Comment:
A boolean prop is automatically `true` if it is set even if it is not explicitly set with `={true}`. So just `<TableStateProvider isNewContext>` sh/would work.

Reply:
🪿